.github/workflows/ci.yml

```diff
@@ -131,9 +131,9 @@ jobs:
           name: packages
           path: /tmp/registry/storage
       # The 2 steps below run npm init @temporalio from a local registry with different samples
-      - run: node scripts/init-from-verdaccio.js --registry-dir /tmp/registry
+      - run: node scripts/init-from-verdaccio.js --registry-dir /tmp/registry --example hello-world
```

Comment:
I'm ok with this rename, just curios about the reason behind it.
I would go with `--sample` because the source repo is called `samples-node`.
Not critical though and I wouldn't delay this PR for it.

Reply:
Not intentional—it was what Next uses. Will change